src/com/ableton/VirtualEnv.groovy

```diff
@@ -96,11 +106,11 @@ class VirtualEnv implements Serializable {
    * @param python Python version or absolute path to Python executable. On Windows, this
    *               should be a Cygwin-style path, but <strong>without the {@code .exe}
    *               extension</strong>, for example: {@code /c/Python27/python}
+   * @param randomSeed If non-zero, use this seed for the random number generator.
    * @see #create(Object, String)
    */
-  VirtualEnv(Object script, String python) {
```

Comment:
In e5190442246eea104e15ffa4a9f5d7648f203cc1, the docstring for the `python` argument should be removed.

Reply:
Fixed in 47c5159.